docs/api/context-bridge.md

```diff
@@ -61,6 +61,20 @@ The `contextBridge` module has the following methods:
 * `apiKey` string - The key to inject the API onto `window` with.  The API will be accessible on `window[apiKey]`.
 * `api` any - Your API, more information on what this API can be and how it works is available below.
 
+### `contextBridge.executeInMainWorld(executionScript)` _Experimental_
+
+<!-- TODO(samuelmaddock): add generics to map the `args` types to the `func` params  -->
+
+* `executionScript` Object
+  * `func` (...args: any[]) => any - A JavaScript function to execute. This function will be serialized which means
```

Comment:
Is the `func` naming here consistent with something else? We can't use `function` because _reserved_ but I think this could be a case where matching `spawn(thing, args)` might be nice. E.g. `exec(fn, args, { ...opts })`

Reply:
This is consistent with the Chrome extension's [ScriptInjection struct](https://developer.chrome.com/docs/extensions/reference/api/scripting#type-ScriptInjection) from `chrome.scripting.executeScript`. I chose to accept an object to also allow this to be easier to [extend in the future.](https://github.com/electron/governance/blob/main/wg-api/best-practices.md#how-will-this-api-be-extended-in-the-future) As an example, we could add a `useGesture` property if it became necessary.

Playwright's [page.evaluate](https://playwright.dev/docs/api/class-page#page-evaluate) is similar to your suggestion except it uses variadic arguments. We'll want to consider extra options though so that's probably not the best.

[ControlledFrame also support script injection.](https://wicg.github.io/controlled-frame/#dom-controlledframe-executescript) It supports an object with `code` and `file` properties.